airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -357,11 +357,11 @@ def request_params(self, stream_state=None, **kwargs):
         if stream_state.get(self.cursor_field):
             issue_worklogs_state = pendulum.parse(stream_state.get(self.cursor_field))
             state_row = int(issue_worklogs_state.timestamp() * 1000)
-            params["startedAfter"] = state_row
+            params[self.cursor_field] = state_row
         return params
 
     def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
-        latest_record_date = pendulum.parse(latest_record.get("started"))
+        latest_record_date = pendulum.parse(latest_record.get("fields", {}).get(self.cursor_field))
```

Comment:
I don't understand why `fields` here?

Reply:
fields is the object that contains our update field